x-pack/plugins/lens/public/editor_frame_service/editor_frame/suggestion_panel.tsx

```diff
@@ -8,6 +8,7 @@ import './suggestion_panel.scss';
 
 import _, { camelCase } from 'lodash';
 import React, { useState, useEffect, useMemo } from 'react';
+import { useDebounce } from 'react-use';
```

Comment:
```suggestion
import useDebounce from 'react-use/lib/useDebounce';
```

to import only that hook 

I'll create an issue to fix this accross kibana and to add a eslint rule

Reply:
https://github.com/elastic/kibana/issues/80181